BookingSpecBindings/Utils.cs

```diff
@@ -0,0 +1,51 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading.Tasks;
+using TechTalk.SpecFlow;
+
+namespace BookingSpecBindings
+{
+
+	class Utils
+	{
+		private static Random random = new Random();
+		public static string rnd(string email)
```

Comment:
It's better to pass to rnd() only int length. This method should not be responsible for parsing strings.